dev.pulsar_edit.Pulsar.json

```diff
@@ -16,6 +16,7 @@
         "--share=network",
         "--device=dri",
         "--filesystem=home",
+        "--filesystem=/media",
```

Comment:
Could you add `/run/media` and `/mnt` for other directories also.

Reply:
finished https://github.com/oneilmw/dev.pulsar_edit.Pulsar/blob/patch-1/dev.pulsar_edit.Pulsar.json
/run/media and /mnt have been added per your request